Student-Class/src/student/pkgclass/Info.java

```diff
@@ -0,0 +1,77 @@
+package student.pkgclass;
+
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+/**
+ *
+ * @author Ziying Guo
+ */
+ public class Info {

```

Comment:
Yes I know but the assignment said use 3 instance family friend and student so I think I should not name it just student and I couldn't think of another name therefore I name the class info.

Sent from my iPhone

> On May 7, 2015, at 11:36, Raffi Khatchadourian <notifications@github.com> wrote:
> 
> In Student-Class/src/student/pkgclass/Info.java:
> 
> > @@ -0,0 +1,77 @@
> > +package student.pkgclass;
> > +
> > +/*
> > + * To change this license header, choose License Headers in Project Properties.
> > + * To change this template file, choose Tools | Templates
> > + * and open the template in the editor.
> > + */
> > +
> > +/**
> > + *
> > + * @author Ziying Guo
> > + */
> > + public class Info {
> Should be: public class Student.
> 
> —
> Reply to this email directly or view it on GitHub.
>